manager/integration/tests/common.py

```diff
@@ -3552,7 +3573,121 @@ def minio_delete_random_backup_block(client, core_api, volume_name):
 
 # TODO: implement nfs_delete_random_backup_block
 def nfs_delete_random_backup_block(client, core_api, volume_name):
```

Comment:
Do we currently limit the backup target to S3? in the test calling code, if so then this is okay.